energy_baselines/requirements.txt

```diff
@@ -0,0 +1,13 @@
+googleapis-common-protos==1.5.8
```

Comment:
I believe quite a few of these are just `pymortar`'s requirements, and aren't generally used. Can you remove the `pymortar` requirements to that those are more flexible to change? I think you can just have scipy, sklearn, scikit-learn, pymortar and pandas.